CefSharp.Wpf/Internals/WpfKeyboardHandler.cs

```diff
@@ -12,19 +12,19 @@ public class WpfKeyboardHandler : IWpfKeyboardHandler
         /// <summary>
         /// The owner browser instance
         /// </summary>
-        private readonly ChromiumWebBrowser owner;
+        protected readonly ChromiumWebBrowser owner;
 
         public WpfKeyboardHandler(ChromiumWebBrowser owner)
         {
             this.owner = owner;
         }
 
-        public void Setup(HwndSource source)
+        public virtual void Setup(HwndSource source)
         {
             // nothing to do here
         }
 
-        public void Dispose()
+        public virtual void Dispose()
```

Comment:
If we merge ImeWpfKeyboardHandler into this existing class, we can avoid changing the `virtual` and `protected` keywords.